cmd/vcert/flags.go

```diff
@@ -694,7 +696,16 @@ var (
 		TakesFile:   true,
 	}
 
-	commonFlags              = []cli.Flag{flagInsecure, flagVerbose, flagNoPrompt}
+	flagUserAgent = &cli.StringFlag{
+		Name: "user-agent",
+		Usage: "Use to set the User-Agent header in HTTP requests to the Venafi REST API. " +
+			"Example: --user-agent example-vcert-script/v0.0.0",
+		Destination: &flags.userAgent,
+		Value:       fmt.Sprintf("%s/%s", UtilityShortName, vcert.GetFormattedVersionString()),
+		EnvVars:     []string{vcertUserAgent},
+	}
```

Comment:
The other flags don't seem to make use of the `EnvVars` field, but it's nice because it helps the user learn about the corresponding env var name in the `--help` output.

Reply:
Good callout, we should look to update the other commands in a future effort.

Reply:
There is reason behind that decision. We need to sit down and think about it. So far the only reason I remember VCert CLI does not use native EnvVar support is to be able to discern the source of the flag (cli, envvar, config file) and print an appropiate message (flag X is overriden by cli...)

Reply:
BTW, Im all in favor to drop the custom env var support we have in favor of the native CLI support.